scripts/get-quality-measures-from-pdfs.js

```diff
@@ -72,19 +72,19 @@ function getRates(path, cb) {
       var remainingFile = data.substring(reportingFound.index + reportingFound[0].length);
       // find the 1, see if a ) or a . comes after each performance rate number
       var firstPos = remainingFile.indexOf('1');
-      var delineationChar = remainingFile[firstPos+1];
+      var delineationChar = remainingFile[firstPos + 1];
 
-      for (var i = 1; i<=numOfRates; i++) {
+      for (var i = 1; i <= numOfRates; i++) {
         // look for the description between 1) and 2)
-        var rateRegex = new RegExp(i + '\\' + delineationChar + '((.|\\s)*?)(' + (i+1)+ '\\' + delineationChar + '|Version 1\\.0)');
```

Comment:
Not sure I understand why the `var` was removed here? Cause the only other place it's defined is above if `(!found)` is `true`, but in that case we won't get to this block of code? Or at least should there be a `let rateRegex` before the if-block and no `var` statements used?